src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/EvaluateGenotypingPerfromanceUnitTests.java

```diff
@@ -0,0 +1,178 @@
+package org.broadinstitute.hellbender.tools.walkers.validation;
+
+import com.google.common.collect.ImmutableMap;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.apache.commons.math3.stat.correlation.PearsonsCorrelation;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+public class EvaluateGenotypingPerfromanceUnitTests extends GATKBaseTest {
+
+    @Test
+    void testPearsonCorrelationAggregator() {
+        final Random rand = new Random(234);
+        final int nEntries = 1000;
+        final double[] x = new double[nEntries];
+        final double[] y = new double[nEntries];
+
+        final EvaluateGenotypingPerformance.PearsonCorrelationAggregator aggregator = new EvaluateGenotypingPerformance.PearsonCorrelationAggregator();
+
+        for (int i = 0; i < nEntries; i++) {
+            x[i] = rand.nextDouble();
+            y[i] = rand.nextDouble();
+
```

Comment:
i dont know the answer to this, but is there a faster/vectorized way of doing this?